server/cluster/coordinator.go

```diff
@@ -453,6 +454,21 @@ func (c *coordinator) resetSchedulerMetrics() {
 	schedulerStatusGauge.Reset()
 }
 
+func collectStoreMetrics(rw, storeAddress, storeLabel string, storeID uint64, getStats func(storeID uint64) (string, *statistics.HotPeersStat)) {
+	typ, stat := getStats(storeID)
```

Comment:
why not pass in `typ` and `stat` directly?

Reply:
because of so much line....

Reply:
Do you think we should revert them?